grc/core/platform.py

```diff
@@ -84,6 +100,8 @@ def load_and_generate_flow_graph(self, file_path, out_path=None, hier_only=False
         try:
             flow_graph = self.make_flow_graph()
             flow_graph.grc_file_path = file_path
+            if self.is_trusted(file_path):
+                flow_graph.view_only = False
```

Comment:
The `viewl_only` attribute seems to be set twice. Here and in `self.make_flow_graph()`. I suspect this one can be removed